### PR TITLE
[Bug] Talkback should shift focus to Snackbar when it displays

### PR DESCRIPTION
**Pull request by phileo99**
## Purpose

* Talkback should shift focus to the error Snackbar when it displays

## Does this introduce a breaking change?

```
[ ] Yes
[ X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ X] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test

* Turn on the Talkback
* Navigate to call setup page.
* Turn off WiFi and attempt to Join call

## What to Check
Verify that the following are valid
* Focus is shifted to Snackbar when it is displayed

### CODE REVIEW

**Review by ShaunaSong — DISMISSED** [review]

**Comment by lpx1989** [comment]
snakbar_error
Based on my testing, when the error snackbar is showing up, talkback focus is on "Go Back" button.

File: azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/fragment/setup/components/ErrorInfoView.kt [hunk 1]
```diff
@@ -45,32 +48,36 @@ internal class ErrorInfoView(private val rootView: View) {
...
+        val errorMessage = getErrorMessage(error, appLocalizationProvider)
 
-        if (errorMessage.isNotEmpty()) {
-            snackBarTextView.text = errorMessage
-            if (!snackBar.isShown) {
-                snackBar.show()
+        if (errorMessage.isBlank()) return
+        snackBarTextView.text = errorMessage
+        snackBar.run {
+            if (isShown) {
```
Comment (Mohtasim) [inline azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/fragment/setup/components/ErrorInfoView.kt:59]: Just to be sure here, if the snackBar was already shown before, why are we dismissing it and showing again? To load the new errorMessage? Or it was a typo?

**Review by Mohtasim — COMMENTED** [review]
Could you please fix the merge conflicts? I think it happened after the refactoring PR got merged in develop/beta.

File: azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/fragment/setup/components/ErrorInfoView.kt (hunk 1 above)
Comment (phileo99) [reply azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/fragment/setup/components/ErrorInfoView.kt:55]: @Mohtasim yes, that is correct.  If there is a snackBar already showing and we have a new non-blank message incoming, we dismiss the snackBar and show it again with the new message.

Comment (Mohtasim) [reply azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/fragment/setup/components/ErrorInfoView.kt:55]: Thanks.

**Review by Mohtasim — DISMISSED** [review]
Looks good to me

**Comment by ShaunaSong** [comment]
Tested.

**Review by ShaunaSong — DISMISSED** [review]

**Review by lpeixin — DISMISSED** [review]

**Review by Mohtasim — APPROVED** [review]
Tested and looks good to me

**Review by ShaunaSong — APPROVED** [review]

**Review by lpeixin — APPROVED** [review]

**Pull request merged**